Update README for clarity

htang085 opened the pull request:
•	Updated the README to clarify the project purpose, how to set up the environment, and how to run the application.
	•	Added a License section referencing the MIT License.
	•	Organized the “Getting Started” instructions into numbered steps.

htang085 committed:
Update README for clarity

dominic-lam reviewed (approved):
LGTM!

colombe-tolokin commented on README.md:
````diff
@@ -1,16 +1,41 @@
+2.	**Set Up the Environment**:
+	```bash
+	conda env create -f environment.yaml
+	conda activate canadian-house-prices-env
````
This should be `canadian_house_prices`

colombe-tolokin commented on README.md:
````diff
@@ -1,16 +1,41 @@
+   ```
+2.	**Set Up the Environment**:
+	```bash
+	conda env create -f environment.yaml
````
the corect file name is `environment.yml`

colombe-tolokin commented:
I also think  we should add a section for `Contributing` and talk a bit about the dataset used,

htang085 committed:
Update README.md

Correct the .yaml file extension name.

htang085 committed:
Update README.md

Add the contributing part to the README.md file

htang085 committed:
Update README.md

Correct the environment name to match what is specified in the environment.yml file.
